src/sdk/python/rtdip_sdk/pipelines/sources/spark/iso/pjm_daily_load_iso.py

```diff
@@ -101,23 +83,24 @@ def _fetch_from_url(self, url_suffix: str, start_date: str, end_date: str) -> by
         logging.info(
             f"Requesting URL - {url}, start_date={start_date}, end_date={end_date}, load_type={self.load_type}"
         )
-        load_key = (
-            "datetime_beginning_ept"
-            if self.load_type != "forecast"
-            else "forecast_datetime_beginning_ept"
-        )
+        
+        load_key = "datetime_beginning_ept"
+
         feed = (
-            "ops_sum_prev_period"
-            if self.load_type != "forecast"
-            else "load_frcstd_7_day"
+            "rt_hrl_lmps"
+            if self.load_type != "day_ahead"
+            else "da_hrl_lmps"
         )
+        
+        print("This feed is being used: ",feed)
```

Comment:
Can we remove all the print statements, if something requires print we need to use logging